packages/storybook/src/nimble/table/table-fit-rows-height-matrix.stories.ts

```diff
@@ -49,9 +56,9 @@ const component = (
         }
     </style>
     <${tableTag}>
```

Comment:
Think we need to set a width on the table for this to be reliable (my monitor was high enough resolution to happen to not show either case with a horizontal scrollbar). Maybe a fixed value like 600px or something?